store/src/java/com/zimbra/cs/filter/FilterUtil.java

```diff
@@ -290,13 +290,23 @@ public static void redirect(OperationContext octxt, Mailbox sourceMbox, MimeMess
             try {
                 outgoingMsg = createRedirectMsgOnError(msg);
                 ZimbraLog.filter.info("Message format error detected.  Wrapper class in use.  %s", e.toString());
+                // ZBUG-3064: If MimeMessage.saveChanges fails, updating header in copy of the message
```

Comment:
looks like redundant code. Create separate bock and use it at both places

Reply:
done